### PR TITLE
Consider separator in checks

### PR DESCRIPTION
**Pull request by vanodevium**
- quote string if contains custom separator ( #22 )
- fixed many typos in files with data for tests

### CODE REVIEW

File: src/helpers/check-special-chars-and-empty.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
-export const checkSpecialCharsAndEmpty = (value) => {
+const SYMBOLS = '\\n\\t,;.\'"`´ ';
```
Comment (vanodevium) [inline src/helpers/check-special-chars-and-empty.js]: @aichbauer If we want to use standard for quotes, how about to use only symbols from standard plus separator symbol?
So, we leave in list:
- double-quote
- comma
- newline symbol

and separator

Comment (aichbauer) [reply src/helpers/check-special-chars-and-empty.js]: @webdevium this is definitely something we should do...

When do we actually have to use double quotes?

Do we need to escape a comma if we have a different separator? 

Maybe:

- double quote (but we have to escape a double quote inside with a leading double quote e.g. "this is a ""test""")
- separator (comma or the used separator for this document (so a comma is not escaped if there is a different separator))
- new lines

What do you think?

Comment (webdevium) [reply src/helpers/check-special-chars-and-empty.js:1]: I will try, but i not understand what I should do with double quote. If on symbol and if few symbols. Can you explain me with some examples, please!

Comment (vanodevium) [reply src/helpers/check-special-chars-and-empty.js]: So, I did everything I could do.
Check my work, please.

**Comment by vanodevium** [comment]
@aichbauer How about validation rules for separator option? It must be one symbol. And, maybe, must be NOT new line or double quote. What do you think?

**Comment by aichbauer** [comment]
@webdevium the validation for the separator option sounds good 👍

**Comment by vanodevium** [comment]
@aichbauer I added a validator and some tests. It looks good to me :)

**Comment by aichbauer** [comment]
@webdevium we'll need to resolve conflicts. I am pretty confident that I will have time to review and merge the PR this weekend if everything is fine.

**Comment by vanodevium** [comment]
Done

**Review by aichbauer — APPROVED** [review]

**Pull request merged**